cmd/export-client/res/configuration-docker.toml

```diff
@@ -0,0 +1,17 @@
+ApplicationName = 'export-client'
+Hostname = '127.0.0.1'
+Port = 48071
+MongoURL = '0.0.0.0'
+MongoUsername = ''
+MongoPassword = ''
+MongoDatabase = 'coredata'
+MongoPort = 27017
+MongoConnectTimeout = 5000
+MongoSocketTimeout = 5000
+ConsulHost ='127.0.0.1'
```

Comment:
Nope ;)

Reply:
will change this follow other modules' configuration

Reply:
cc @drasko , please review again.